assembly/pom.xml

```diff
@@ -30,6 +30,11 @@
             <artifactId>hbase_sql-${envClassifier}</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-core_${scala.binary.version}p</artifactId>
+            <version>${spark.version}</version>
```

Comment:
Would this add spark classes in splice uber jar?

Reply:
It didn't seem to, but it wasn't doing anything really so I removed it. Thanks